src/Microsoft.PowerApps.TestEngine/SingleTestRunner.cs

```diff
@@ -94,20 +95,26 @@ public async Task RunTestAsync(string testRunId, string testRunDirectory, TestSu
 
                 Logger.LogInformation($"\n\n---------------------------------------------------------------------------\n" +
                     $"RUNNING TEST SUITE: {testSuiteName}" +
-                    $"\n---------------------------------------------------------------------------\n\n" +
-                    $"Browser configuration: {JsonConvert.SerializeObject(browserConfig)}");
-
+                    $"\n---------------------------------------------------------------------------\n\n");
+                Logger.LogTrace($"Browser configuration: {JsonConvert.SerializeObject(browserConfig)}");
 
                 // Set up test infra
                 await _testInfraFunctions.SetupAsync();
-                Logger.LogInformation("Test infrastructure setup finished");
+                Logger.LogDebug("Test infrastructure setup finished");
+
+                desiredUrl = _urlMapper.GenerateTestUrl(domain, queryParams);
+                Logger.LogInformation($"Desired URL: {desiredUrl}");
 
-                var desiredUrl = _urlMapper.GenerateTestUrl(domain, queryParams);
-                Logger.LogTrace($"Desired URL: {desiredUrl}");
+                var startString = $"\nRunning test suite: {testSuiteName}" +
+                $"\n\tTest results will be stored in: {testRunDirectory}" +
+                $"\n\tBrowser: {browserConfigName}" +
+                $"\n\tApp URL: {desiredUrl.Replace("&source=testengine", String.Empty)}";
+
+                Console.Out.WriteLine(startString);
 
                 // Navigate to test url
                 await _testInfraFunctions.GoToUrlAsync(desiredUrl);
-                Logger.LogInformation("Successfully navigated to target URL");
```

Comment:
Wouldn't this be still better. In default log level setting we won't have this logged with the new change.
I suggest keeping it as it is, unless there is a reason to do so.